ocs_ci/ocs/constants.py

```diff
@@ -1504,7 +1517,7 @@
 
 # Values from configmap noobaa-postgres-config
 NOOBAA_POSTGRES_TUNING_VALUES = """
-max_connections = 300
+max_connections = 600
```

Comment:
This will need changes based on the version too. i.e., until 4.9, it would be 300 and from 4.10, it would be 600.

Reply:
Ohh okay, will do it.